readme.prod.md

```diff
@@ -0,0 +1,21 @@
+## Keating
```

Comment:
I would like to rename this file from `readme.prod.md` to something like `production_deployment.md` or similar. So the filename will be related with content